phpBB/phpbb/search/fulltext_native.php

```diff
@@ -1202,7 +1201,7 @@ public function author_search($type, $firstpost_only, $sort_by_sql, $sort_key, $
 								$post_visibility
 								$sql_fora
 								AND t.topic_id = p.topic_id
-								$sql_time" . (($this->db->get_sql_layer() == 'sqlite' || $this->db->get_sql_layer() == 'sqlite3') ? ')' : '');
+								$sql_time" . (($this->db->get_sql_layer() == 'sqlite3') ? ')' : '');
```

Comment:
all parenthesis are useless: `$sql_time" . $this->db->get_sql_layer() == 'sqlite3' ? ')' : '';`